src/counters.c

```diff
@@ -802,6 +814,9 @@ static int StatsOutput(ThreadVars *tv)
         }
     }
 
+    /* sort stats table */
+    qsort(stats_table.stats, stats_table.nstats, sizeof(StatsRecord), StatsTableRecordsSortByName);
```

Comment:
Is there a way to avoid sorting the stats each time they're being prepared for output?

Reply:
Good question. I'll investigate that further. Thanks!!

Reply:
I discussed this with Victor, and he thinks that trying it at the source would require major refactoring, so I guess we'll be leaving this as is, for now :P